index.ios.js

```diff
@@ -113,9 +121,9 @@ export default class fireball extends Component {
         // DO SOMETHING WITH RESPONSE
         console.log('res: ', res)
         this.setState({
-          score: res.data.responses[0].labelAnnotations[0].score,
+          score: Math.floor(res.data.responses[0].labelAnnotations[0].score * 100),
```

Comment:
Useful library that does this: 

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toFixed